src/content/core/okteto-variables.mdx

```diff
@@ -197,13 +197,34 @@ USERNAME has a static value, and PASSWORD is dynamically generated.
-- `OKTETO_COMPOSE_VOLUME_AFFINITY_ENABLED`:
-- `OKTETO_KUBERNETES_TIMEOUT`:
-- `OKTETO_HOME`:
-- `OKTETO_FORCE_REMOTE`:
-- `OKTETO_FOLDER`:
-- `OKTETO_REMOTE_CLI_IMAGE`:
+- `OKTETO_COMPOSE_VOLUME_AFFINITY_ENABLED`: Compose services mounting the same volume will be placed on the same node using Kubernetes's pod affinity. (Defaults to `true`)
+- `OKTETO_KUBERNETES_TIMEOUT`: specifies the timeout while deploying. (defaults to `0`)
+- `OKTETO_FORCE_REMOTE`: if true deploy the repository on the cluster vs locally.
+- `OKTETO_FOLDER`: defines the path where the okteto folder is located (defaults to `$OKTETO_HOME/.okteto`).
+- `OKTETO_REMOTE_CLI_IMAGE`: specifies the CLI image that Okteto is going to use while deploying/destroying on remote.
 - `OKTETO_LOCAL_REGISTRY_STORE_ENABLED`: enables the local registry credentials over the credentials stored in the cluster. (Defaults to `true`) 
+- `OKTETO_BIN`: version of the CLI that the init containers will use.
+- `OKTETO_SKIP_CLEANUP`: if set skips the deletion of the syncthing folder when running `okteto down`
+- `OKTETO_USE_STATIC_KUBETOKEN`: use a fixed token rather than a dynamic token for interactions with the Kubernetes API. (Defaults to `false`)
+- `OKTETO_HOME`: defines the path of okteto folder (Defaults to `$HOME`)
+- `OKTETO_URL`:  specifies the Okteto cluster URL to connect to.
+- `OKTETO_TOKEN`: specifies the Okteto token that should be used to connect to the Okteto cluster.
+- `OKTETO_CONTEXT`: specifies the Kubernetes cluster name to connect to.
+- `OKTETO_NAMESPACE`: specifies the Kubernetes namespace that the CLI will operate.
+- `COMPOSE_FILE`: specifies the compose file that the user has to use.
+- `OKTETO_AUTOGENERATE_STIGNORE`: if true generates the `.stignore` file when running `okteto up`. (Defaults to `false`)
+- `OKTETO_AUTODEPLOY`: if set forces the deployment of the development environment on `okteto up` (Defaults to `false`)
+- `OKTETO_DEPOT_TOKEN`: specifies the Depot token that the build is going to use.
+- `OKTETO_DEPOT_PROJECT_ID`: specifies the Depot project ID where the image will be built.
+- `KUBECONFIG`: specifies where the kubeconfig is stored (several files can be set by separating them by `:` on unix systems and `;` on windows)
+- `BUILDKIT_PROGRESS`: defines the output progress that buildkit must show.
+- `OKTETO_COMPOSE_UPDATE_STRATEGY`: defines the update strategy that the compose must translate (it can be one of: `rolling`/`recreate`/`on-delete`)
+- `OKTETO_DISABLE_SPINNER`: if set disables the spinner rotation (defaults to `false`)
+- `OKTETO_K8S_REQUESTS_LOGGER_ENABLED`: If true logs all requests to the kubernetes API. (defaults to `true`)
+- `OKTETO_SYNCTHING_VERSION`: specifies the syncthing version the CLI must use
+- `OKTETO_EXECUTE_SSH`: specifies if the command must be executed through SSH vs Kubernetes exec (defaults to `true`)
```

Comment:
Let's not document this. Instead, let's create an issue to remove the logic for k8s exec and always use SSH